tmuxp/cli.py

```diff
@@ -10,6 +10,7 @@
 import logging
 import os
 import sys
+from pathlib import Path
```

Comment:
Will this work on python 2.7? 

I think we'll do one final release for Python 2.7. If there is a way to do this without `pathlib` that'd be worth checking